docs/maintainers/docker.md

```diff
@@ -0,0 +1,29 @@
+# Docker
+
+## Context
+
+Github Actions should automatically build and publish a Docker image for each release. If Github Actions failed to build or publish the Docker image, you can manually build and publish a Docker image using this guide.
```

Comment:
I think we could condense this

Reply:
Do you have a suggestion? Would you prefer this?

```suggestion
Github Actions should automatically build and publish a Docker image for each release. If Github Actions failed, you can manually build and publish a Docker image using this guide.
```